pkg/config/database.go

```diff
@@ -31,9 +35,28 @@ func (d *Database) Open(logger *zap.SugaredLogger) (*icingadb.DB, error) {
 		driver.Register(logger)
 	})
 
-	dsn := fmt.Sprintf(
-		"%s:%s@tcp(%s:%d)/%s?timeout=60s",
-		d.User, d.Password, d.Host, d.Port, d.Database)
+	myCfg := mysql.NewConfig()
```

Comment:
I would go with `config: = mysql.Config {...}`. The default values of `NewConfig()` are literally of no use, and `myCfg` doesn't read as well as just` config`.

Reply:
## Use

### DSN w/ `config := &mysql.Config{}`

```
idb:idb@tcp(127.0.0.1:6033)/idb?allowNativePasswords=false&checkConnLiveness=false&timeout=1m0s&maxAllowedPacket=0
```

### DSN w/ `config := mysql.NewConfig()`

```
idb:idb@tcp(127.0.0.1:6033)/idb?timeout=1m0s
```

Reply:
You're right. My bad. Let's stick to the defaults.